packages/react-components/react-accordion/src/components/Accordion/Accordion.tsx

```diff
@@ -5,6 +5,7 @@ import { useAccordionContextValues_unstable } from './useAccordionContextValues'
 import type { AccordionProps } from './Accordion.types';
 import type { ForwardRefComponent } from '@fluentui/react-utilities';
 import { useAccordionStyles_unstable } from './useAccordionStyles';
+import { useCustomStyleHooks_unstable } from '@fluentui/react-shared-contexts';
```

Comment:
_nit:_ Keep `import type` statements at the bottom

```suggestion
import { useCustomStyleHooks_unstable } from '@fluentui/react-shared-contexts';
import { useAccordionStyles_unstable } from './useAccordionStyles';
import type { AccordionProps } from './Accordion.types';
import type { ForwardRefComponent } from '@fluentui/react-utilities';
```